package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"no test specified\" && exit 0",
-    "start": "node index.js"
+    "start": "node index.js",
+    "lint": "eslint index.js ./models/ ./routes/ ./config/"
```

Comment:
just do `eslint .` and it'll lint everything, then create an ignore file and add files you don't plan to lint in it